src/canisters/frontend/ic-asset/src/evidence/mod.rs

```diff
@@ -37,11 +40,21 @@ pub async fn compute_evidence(
     let asset_descriptors = gather_asset_descriptors(dirs, logger)?;
 
     let canister_assets = list_assets(canister).await?;
+    info!(
+        logger,
+        "Fetching properties for all assets in the canister."
+    );
+    let canister_asset_properties = get_assets_properties(canister, &canister_assets).await?;
 
```

Comment:
Please display another info! line here, because the next step takes some time too